Sources/BeeTracking/Visualizations.swift

```diff
@@ -77,6 +77,50 @@ public func plotPatchWithGT(frame: Tensor<Float>, actual: Pose2, expected: Pose2
   return (fig, ax)
 }
 
+public func plotPoseDifference(track: [Pose2], withGroundTruth expected: [Pose2], on ax: PythonObject) {
+  var thetaDiff = zip(track, expected).map{pow(($0.0.rot.theta - $0.1.rot.theta), 2.0)}
+  var posDiff = zip(track, expected).map{pow(($0.0.t.x - $0.1.t.x), 2.0) + pow(($0.0.t.y - $0.1.t.y), 2.0)}
+  ax.plot(thetaDiff, posDiff)
+  ax.set_title("L2 Theta Difference (X-axis) vs. L2 X, Y Difference Over Time")
+}
+
+/// plot Comparison image
+public func plotFrameWithPatches(frame: Tensor<Float>, actual: Pose2, expected: Pose2, firstGroundTruth: Pose2) -> (PythonObject, PythonObject) {
+  let plt = Python.import("matplotlib.pyplot")
+  let mpl = Python.import("matplotlib")
+
+  let (fig, ax) = plt.subplots(figsize: Python.tuple([8, 4])).tuple2
+  ax.imshow(frame.makeNumpyArray() / 255.0)
+  let actualBoundingBox = OrientedBoundingBox(center: actual, rows: 40, cols: 70)
+  ax.plot(Python.list(actualBoundingBox.corners.map{$0.x} + [actualBoundingBox.corners.first!.x]), Python.list(actualBoundingBox.corners.map{$0.y} + [actualBoundingBox.corners.first!.y]), "r-")
```

Comment:
I think you don't need to explicitly convert to Python.list here, but anyway.

Reply:
Ok I'll update this!

Reply:
@ProfFan removed the call to Python.list

Reply:
Nice :)